ImageChecks.ps1

```diff
@@ -0,0 +1,141 @@
+param(

```

Comment:
please follow the PowerShell guidelines from https://github.com/ader1990/PowerShell-Guidelines

Reply:
I meant # **ALL** the rules from in there, starting with putting all the main logic inside one Main like function. 
Those rules are mandatory and not recommendations.
In your case it can be named:

Validate-Image